openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -138,58 +138,53 @@
 - control_key: AU-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - key: a
-      text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+      text: |      
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
+ 
     - key: b
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-161
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-162
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
 
 - control_key: AU-5 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: complete
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-163
+        Satisfaction of this control is inherited from configuration
+        of the underlying operating system.
```

Comment:
When cluster logging is enabled, to meet AU-5(2), we'll need to document how to setup alerts to system owners upon failure conditions. Believe there is no way to do this currently, hence the open Jira with the OpenShift product team.

Control link: https://nvd.nist.gov/800-53/Rev4/control/AU-5#enhancement-2

text:

AU-5(2) | RESPONSE TO AUDIT PROCESSING FAILURES \| REAL-TIME ALERTSThe information system provides an alert in [Assignment: organization-defined real-time period] to [Assignment: organization-defined personnel, roles, and/or locations] when the following audit failure events occur: [Assignment: organization-defined audit failure events requiring real-time alerts].
-- | --